ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -944,24 +974,22 @@ def test_confusion_matrix(
         predictions=predictions,
         num_classes=num_classes,
     )
+    
 
-
```

Comment:
It seems that in adding `logical_not` here you have removed the test for `polyval`

Reply:
This would need to be reimplemented :)